commercetools-test-lib/src/main/java/io/sphere/sdk/test/CompletableFutureCallbackAdapter.java

```diff
@@ -0,0 +1,31 @@
+package io.sphere.sdk.test;
+
+
+
+import io.sphere.sdk.models.Base;
+import org.apache.http.concurrent.FutureCallback;
+
+import java.util.concurrent.CompletableFuture;
+
+final class CompletableFutureCallbackAdapter<T> extends Base implements FutureCallback<T> {
```

Comment:
I would suggest to add some javadoc that explains why we had to duplicate this class. What do you think?